src/components/common/ColonyActions/helpers/getActionTitleValues.ts

```diff
@@ -104,6 +106,12 @@ const getMessageDescriptorKeys = (actionType: AnyActionType) => {
         ActionTitleMessageKeys.Recipient,
         ActionTitleMessageKeys.Initiator,
       ];
+    case actionType.includes(ExtendedColonyActionType.UpdateAddressBook):
```

Comment:
this is not necessary, maybe we can reuse the [strings](https://github.com/JoinColony/colonyCDapp/blob/bcf30454ca410212642c4c0b54e0fcb8880987f4/src/components/common/ColonyActions/helpers/getActionTitleValues.ts#L125) implemented for displaying titles in the action table?